test/fixture/runner/agreeable.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Was this necessary for something?

Reply:
Cops for Ruby >= 2.3 require this frozen string literal comment. Since I thought we were bumping `standard`'s min ruby version to 2.3, adding this for tests made sense to me.

Reply:
Let's definitely disable whatever cop is enforcing this — one possibility based on the above comment about performance being on-by-default, this got re-turned-on when it was intentionally turned off earlier

Reply:
Ok sounds good!

Reply:
I'm having an issue making this fixture and it's related test pass. The test that is failing is: 

```
  1) Failure:
Standard::Runners::RubocopTest#test_empty_output_on_quiet_success [/Users/jakeyesbeck/standard/test/standard/runners/rubocop_test.rb:20]:
--- expected
+++ actual
@@ -1 +1,8 @@
-""
+"== test/fixture/runner/agreeable.rb ==
+" +
+"C:  1:  1: Style/FrozenStringLiteralComment: Missing magic comment # frozen_string_literal: true.
+" +
+"
+" +
+"1 file inspected, 1 offense detected
+"
```

The test fails on the frozen string cop despite me adding to `config/base.yml`:

```
Style/FrozenStringLiteralComment:
  Enabled: false
```

@searls any ideas?